Tests/SonarQube.TeamBuild.PreProcessor.Tests/AnalysisConfigGeneratorTests.cs

```diff
@@ -22,6 +23,61 @@ public class AnalysisConfigGeneratorTests
         #region Tests
 
         [TestMethod]
+        public void AnalysisConfGen_Simple()
```

Comment:
There wasn't a basic test of the config file generator (just specialised tests to check that sensitive data isn't written out).